config/application.yml

```diff
@@ -10,4 +10,4 @@
 #   stripe_api_key: sk_live_EeHnL644i6zo4Iyq4v1KdV9H
 #   stripe_publishable_key: pk_live_9lcthxpSIHbGwmdO941O1XVU
 
-PLANT-API-KEY: 117050ef67998cb1747b93f65d4288f1
+PLANT-API-KEY: 'cfe4176a8348418b80adac25bccd5d54'
```

Comment:
I have config in gitignore file so not sure why this is pushed up. I'll look into it.